apps/astarte_appengine_api/priv/static/astarte_appengine_api.yaml

```diff
@@ -292,7 +292,7 @@ paths:
       tags:
         - device
       summary: Update a device writeable property
-      description: Update any of the writeable device properties such as device aliases or device metadata.
+      description: Update any of the writeable device properties such as device aliases, device attributes or credentials inhibited.
```

Comment:
is `credentials inhibited` no longer writeable?

Reply:
The opposite, this documentation was not updated when `credentials_inhibited` was made writable so I fixed that